src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpAuthHelper.cs

```diff
@@ -272,6 +272,21 @@ public void SaveServerCredentialsToCache(Uri uri, uint authScheme, ICredentials
             }
         }
 
+        public void ChangeDefaultCredentialsPolicy(SafeWinHttpHandle requestHandle, bool allowDefaultCredentials)
+        {
+            uint optionData = allowDefaultCredentials ?
+                Interop.WinHttp.WINHTTP_AUTOLOGON_SECURITY_LEVEL_LOW :
```

Comment:
Shouldn't we instead use `WINHTTP_AUTOLOGON_SECURITY_LEVEL_MEDIUM` which allows credentials to be sent only to intranet sites.

Using `WINHTTP_AUTOLOGON_SECURITY_LEVEL_LOW` will allow NTLM hashes to be sent to Internet-based servers which can potentially find the password from the hash.